disperser/dataapi/server_v2.go

```diff
@@ -105,13 +105,13 @@ func (s *ServerV2) Start() error {
 	{
 		blob := v2.Group("/blob")
 		{
-			blob.GET("/blob/feed", s.FetchBlobFeedHandler)
-			blob.GET("/blob/:blob_key", s.FetchBlobHandler)
+			blob.GET("/blobs/feed", s.FetchBlobFeedHandler)
+			blob.GET("/blobs/:blob_key", s.FetchBlobHandler)
 		}
 		batch := v2.Group("/batch")
 		{
-			batch.GET("/batch/feed", s.FetchBatchFeedHandler)
-			batch.GET("/batch/:batch_header_hash", s.FetchBatchHandler)
+			batch.GET("/batches/feed", s.FetchBatchFeedHandler)
+			batch.GET("/batches/:batch_header_hash", s.FetchBatchHandler)
 		}
 		operators := v2.Group("/operators")
```

Comment:
Or we make it `/operator/...` if it's preferred

Reply:
Think plural is better since the response is for multiple operators right?

Reply:
It can be for a single operator as well. I think it's like "this is API for dealing with multi operators"  rather than saying it'll just be returning one or multiple